src/components/graph-view.js

```diff
@@ -184,8 +188,10 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   componentDidMount() {
     const { initialBBox, minZoom, maxZoom } = this.props;
 
-    // TODO: can we target the element rather than the document?
-    document.addEventListener('keydown', this.handleWrapperKeydown);
+    if (this.props.customKeyEvents) {
```

Comment:
customKeyEvents flag for enablement of events of graph-view, 
to override key combinations in consumers, do we need to provide customKeyEvents as false and handle locally in consumer component?

Reply:
I think this prop may be named incorrectly. I intended for it, if passed in, to disable the custom key handlers that the graph-view is creating.

I think I will rename this to something like: "disableGraphViewKeyHandlers" and set it to false by default but if true, we will not add the listeners.

From our library, we will set it to true and handle key events ourselves.

Reply:
cool cool sounds good